packages/components/src/components/hds/app-side-nav/base.hbs

```diff
@@ -4,6 +4,8 @@
 }}
 {{! IMPORTANT: we need to add "squishies" here (~) because otherwise the whitespace added by Ember causes the empty element to still have visible padding - See https://handlebarsjs.com/guide/expressions.html#whitespace-control }}
 <div class="hds-app-side-nav" ...attributes>
+  <h2 class="sr-only">Local navigation</h2>
```

Comment:
Let me know if this header should use different wording, be moved elsewhere in the DOM, and/or have the text exposed in case consumers would want to override it, etc.

Reply:
@shleewhite @MelSumner

Reply:
Should this be "Application navigation" ? 

Also, is there an aria-label on the nav element in the sidebar? It might be redundant to have both; I'd update in these ways:

1. Add an `id` to the `h2`
2. change the `aria-label` in the aside nav to be `aria-labelledby` and give it the value of the `h2`'s `id` attribute.

Reply:
The `nav` element which wraps each list doesn't have an aria-label (I could add an aria-labelledby though/). There can be multiple `nav` elements potentially and likely. The `appFrame` `sidebar` child component where the `AppSideNav` slots into doesn't have an aria-label.

Does it matter if the h2 labels multiple nav elements?

Reply:
I updated the h2 text to "Application local navigation" since the AppSideNav is supposed to contain page level navigation and to be paired with the AppHeader which contains "global" & "utility" navigation. Let me know if I should change the text to something else again.

I added a hard-coded id to the h2 and aria-labelledby with the id valu to the `nav` elements wrapping lists of links. Again there are likely to be multiple nav elements. (I hard-coded the ids instead of generating since it didn't seem like we would really need to generate them but I can change that too if desired.)

Please let me know if these changes are ok or if I should make more. @MelSumner @shleewhite @alex-ju

Reply:
I think its fine.... does the AppHeader navigation have an accessible name?

Reply:
If the AppHeader needs to have a header or aria-label or similar added could you please open a new issue or PR? I want to focus on the AppSideNav for the current open PRs.